packages/config/src/Config.js

```diff
@@ -448,8 +450,18 @@ function inferWebStartupImages(config: Object = {}, getAbsolutePath: Function, o
 
   let splashImageSource;
   if (webSplash.image || iOSSplash.image || splash.image) {
+    const resizeMode =
+      webSplash.resizeMode || iOSSplash.resizeMode || splash.resizeMode || 'contain';
+    const backgroundColor =
+      webSplash.backgroundColor ||
+      iOSSplash.backgroundColor ||
+      splash.backgroundColor ||
```

Comment:
Is this logic correct? Given this configuration:
{
  splash: {
    backgroundColor: 'black'
  },
  ios: {
    splash: {
      backgroundColor: 'white'
    }
  }
}
```
I'd expect web to use the default `splash` configuration because platform specific `web` configuration is not specified. However, it seems it will use the iOS specific configuration instead.

I think this code should not use the `ios` splash configuration at all.

Reply:
I read an outdated document about iOS being the only platform to use the background_color value. Looks like Google does too: https://developers.google.com/web/tools/lighthouse/audits/custom-splash-screen